draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -4740,7 +4744,8 @@ Content-Encoding: gzip
 <t>
    An origin server &SHOULD; verify that the PUT representation is
    consistent with any constraints the server has for the target
-   resource that cannot or will not be changed by the PUT.  This is
+   resource, in that the constraints cannot or will not be changed
+   by the PUT.  This is
```

Comment:
It feels weird to me for there to be "constraints" that sometimes constrain the acceptable input and sometimes can be modified by the input -- the latter doesn't feel like much of a "constraint".